pom.xml

```diff
@@ -24,7 +24,7 @@
 
   <properties>
     <rhba.images.version.prefix>74</rhba.images.version.prefix>
-    <rhba.images.branch>master</rhba.images.branch>
+    <rhba.images.branch>7.4.x</rhba.images.branch>
```

Comment:
Please revert this line

Reply:
Set back to master and update version prefix to 75